projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/complex/read-complex-field-table.html

```diff
@@ -1,6 +1,6 @@
 <div class="complex-panel">
-  <dl class="complex-panel-title"><dt><span class="text-16">{{caseField.label | rpxTranslate}}</span></dt><dd></dd></dl>
-  <table class="complex-panel-table" aria-describedby="complex field table">
+  <dl class="complex-panel-title"><dt><span id="complex-table" class="text-16">{{caseField.label | rpxTranslate}}</span></dt><dd></dd></dl>
```

Comment:
And again. Suggest you use "complex-title" + this.caseField.id()

Reply:
Done